File: bankingtool.client/src/app/dto/request/bank-account/get-account-type-drop-down-list-request.ts
import { RequestId } from "../request-id";

export interface GetAccountTypeDropDownListRequestObject {
  request: GetAccountTypeDropDownListRequest;
}

export interface GetAccountTypeDropDownListRequest {
  customer: GetAccountTypeDropDownListRequestCustomer;
  bank: GetAccountTypeDropDownListRequestBank;
}

export interface GetAccountTypeDropDownListRequestCustomer extends RequestId {

}

export interface GetAccountTypeDropDownListRequestBank extends RequestId {

}

File: BankingTool.Model/Dto/RequestDto/BankAccount/GetAccountTypeDropDownListRequest.cs
using BankingTool.Model.Dto.RequestDtos;

namespace BankingTool.Model.Dto.RequestDto.BankAccount
{
    public class GetAccountTypeDropDownListRequestObject
    {
        public GetAccountTypeDropDownListRequest Request { get; set; }
    }
    public class GetAccountTypeDropDownListRequest
    {
        public GetAccountTypeDropDownListRequestCustomer Customer { get; set; }
        public GetAccountTypeDropDownListRequestBank Bank { get; set; }
    }
    public class GetAccountTypeDropDownListRequestCustomer : RequestId
    {
    }
    public class GetAccountTypeDropDownListRequestBank : RequestId
    {
    }
}
